src/pages/login/Login.test.js

```diff
@@ -11,7 +12,11 @@ describe("Login Page", () => {
     });
     const {loginWithRedirect} = useAuth0();
 
-    render(<Login />);
+    render(
+      <BrowserRouter>
+        <Login />
```

Comment:
render method provides wrapper as well if the component is wrapped like this **render(<Login/>, {wrapper: BrowserRouter});**

Reply:
I hope we can use any of the ways to wrap the component, both are similar.